src/actions/updateArticleAction.js

```diff
@@ -0,0 +1,15 @@
+import axiosInstance from '../services/requestHandler';
```

Comment:
like innocent said you might want to compile the article related actions into one file